avd_docs/aws/autoscaling/AVD-AWS-0340/docs.md

```diff
@@ -0,0 +1,13 @@
+
+Ensures all autoscaling groups contain at least 1 instance..
```

Comment:
```suggestion
Ensures all autoscaling groups contain at least 1 instance.
```

Reply:
You'll probably need to fix this in the rule itself and not in the generated markdown.